docs/sandbox/apps/tqm.md

```diff
@@ -0,0 +1,165 @@
+# tqm
+
+## What is it?
+
+[tqm](https://github.com/l3uddz/tqm){: target=_blank rel="noopener noreferrer" } is a CLI tool to manage your torrent client queues. Primary focus is on removing torrents that meet specific criteria.
+
+- The tqm binary is downloaded and a service and timer file created when the config is identified.
+
+!!! Note
+      📢 You will need to have `config.yaml` in place (`/opt/tqm/`) for the role to run successfully.  [Here](https://github.com/l3uddz/tqm#example-configuration) is an example config you can grab and fill in with your own details.
+
+| Details     |
+|-------------|
+| [:octicons-mark-github-16: Github](https://github.com/l3uddz/tqm){: .header-icons target=_blank rel="noopener noreferrer" }|
+
+Recommended install types: Feederbox, Saltbox, Core
+
+### 1. Setup
+
+Edit in your favorite code editor  (with json highlighting) or even a unix editor like nano.
```

Comment:
JSON?

Reply:
Oh whoops. damnit. I stole it from cloudplow.md

Reply:
Fixed. going to add a bit about adjusting the service file. and changing the settings.yml file